src/tests/ie_test_utils/common_test_utils/test_common.cpp

```diff
@@ -61,9 +65,18 @@ inline size_t getVmSizeInKB() {
 
 TestsCommon::~TestsCommon() {
     InferenceEngine::executorManager()->clear();
+
+#ifdef ENABLE_CONFORMANCE_PGQL
+    delete PGLink;
```

Comment:
Why don't you use smart pointer?

Reply:
On Linux I see issues while compilation in case I use smart pointer and struct isn't defined. And I didn't want to bring here custom smart pointer implementation.